about.html

```diff
@@ -0,0 +1,119 @@
+                        while enjoying your time with friends and family.</p>
+                        <p>
+                            If you love camping, then come and join our friendly community of campers. See what other people are doing on the site and support them!
+                        </p>
+                </div>
+                <div class="sec1-pt3">
+                    <p>Please contact us per Email for any further questions</p>
+                    <p>about All In Campsters</p>
+                    <p>allincampsters@camper.com</p>
+                </div>
+            </div>
+        </section>
+
+        <section id="sec2">
+            <div class="sec2-wrapper">
+                <div class="sec2-pt1">
+                    <h2>All In Campsters</h2>
+                    <h2>Simple and direct logo</h2>
+                    <span>____</span>
+                </div>
+                <div class="sec2-pt2">
+                    <p>The logo of All In Campsters was decided</p>
+                    <p>through the logo competition from 8. June to 7. July</p>
+                    <p>We chose our logo for two main reasons. The first and most important one is that we enjoy working in a logo. The second reason is that we want our users to feel comfortable when they visit our website and interact with all the information on it.</p>
+                </div>
+                <div class="sec2-pt3">
+                    <img src="./assets/All In Campsters.svg" alt="logo">
+                </div>
+            </div>
+        </section>
+
+        <section id="sec3">
+            <div class="sec3-wrapper">
+                <div class="sec3-title">
+                    <h2>See the All In Campsters Blog</h2>
+                    <span>____</span>
+                </div>
+                <div class="sec3-desc">
+                    <p>Take a look at the last two All In Campsters blog posts</p>
+                </div>
+                <div class="sec3-blog">
+                    <span>
+                        <h3 class="blog-title-1"><span>Feb</span><br>Romantic Valentine's Day Destinations</h3>
+                        <img src="./assets/camp-blog-1.jfif" class="blog-img-1" alt="Blog Post 1">
+                    </span>
+                    <span>
+                        <h3 class="blog-title-2"><span>Jan</span><br>Campground Safety: A Guide for campers</h3>
+                        <img src="./assets/camp-blog-2.jpg" class="blog-img-2" alt="Blog Post 2">
+                    </span>
+                </div>
+            </div>
+        </section>
+        <section id="partner">
+            <div class="partner-title">
+                <h3>Main Program</h3>
+                <span>______</span>
+            </div>
+            <div class="partners">
+
+            </div>
+        </section>
+        <footer>
+            <div class="copy">&copy;All In Campsters 2023</div>
+        </footer>
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly `hide` your `Mani Program` from About page's desktop version because [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) it available only **mobile version** and give correct `background color` for footer for **desktop version** 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/217055558-dcbc2c8b-62ff-4145-8787-935ea5c35705.png)